XpressAI/vecto-python-sdk
# Init python SDK with wheel and tests

First 
To Test:
- **Build wheel**:  Would recommend building it in a venv. If so, you'll need to `pip install wheel`. Then build the wheel via `python setup.py bdist_wheel --universal`, verify it's successful.
- **Test pytests**: currently the tests are ported from the [vecto api tests repository](https://github.com/XpressAI/vecto-api-test). To run them, export `user_token`, `public_token`, and `vector_space_id` as variables from the [.env](https://github.com/XpressAI/vecto-api-test/blob/main/vecto_config.env). You will also need to install the test packages to run them.
```
pip install -r vecto/api-tests/requirements.txt
pytest vecto/api-tests/test_user.py
pytest vecto/api-tests/test_public.py
```

Todo in next PRs:
- setup github action to run tests
- figure out why the public tests have failing tests
